src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/ModelBinderDescriptorExtensions.cs

```diff
@@ -80,5 +80,22 @@ public static class ModelBinderDescriptorExtensions
             descriptors.Insert(index, descriptor);
             return descriptor;
         }
+
+        /// <summary>
+        /// Adds a <see cref="TypeSpecificModelBinder"/> to a descriptor collection.
+        /// </summary>
+        /// <param name="descriptors">A list of ModelBinderDescriptors.</param>
+        /// <param name="type">The model type that the given instance should bind with.</param>
+        /// <param name="modelBinder">An <see cref="IModelBinder"/> instance.</param>
+        /// <returns>The <see cref="ModelBinderDescriptor"/> representing the added instance.</returns>
+        public static ModelBinderDescriptor Add([NotNull] this IList<ModelBinderDescriptor> descriptors,
+                                                [NotNull] Type type,
+                                                [NotNull] IModelBinder modelBinder)
```

Comment:
should we consider any of the other things that registering a model binder allows you to do? ModelBinderProvider, type-activated model binder